src/app/choose-template/page.tsx

```diff
@@ -0,0 +1,60 @@
+'use client' // we need this to be a client component because nhsuk-react-components uses client-only react features
+import { Radios, Button, ErrorSummary, ErrorMessage,  } from 'nhsuk-react-components';
+import { useFormState } from 'react-dom';
+import { chooseTemplateServerAction, FormState } from './choose-template-server-action';
+import { chooseTemplatePageContent } from '../../content/content';
+
+export default function ChooseTemplatePage() {
+    const { pageHeading, errorHeading, options } = chooseTemplatePageContent;
+    const [state, action] = useFormState<FormState, FormData>(chooseTemplateServerAction, { formErrors: [], fieldErrors: {}});
+    console.log(state);
+    console.log(Object.entries(state.fieldErrors));
```

Comment:
Is there a reason why we are keeping console.log in the code. I think it's best to not include it.